scene/gui/color_picker.cpp

```diff
@@ -1078,7 +1078,7 @@ void ColorPicker::_palette_file_selected(const String &p_path) {
 #endif
 		} break;
 		case FileDialog::FileMode::FILE_MODE_SAVE_FILE: {
-			ColorPalette *palette = memnew(ColorPalette);
+			Ref<ColorPalette> palette = memnew(ColorPalette);
```

Comment:
Generally the preferred style is
```suggestion
			Ref<ColorPalette> palette;
			palette.instantiate();
```
Some code just wrongly uses RefCounted.

Reply:
Makes sense.
In the context of this change, it would also be possible to disallow `memnew` for `RefCounted` types (so callers _have_ to use `.instatiate()` instead). I suppose now would be a good time to figure out which style we prefer going forward.